TypeCobol/Compiler/Sql/Model/Information.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.Sql.Model
+{
+    public abstract class Information : SqlObject
+    {
+        public abstract InformationType Type { get; }
+        protected override bool VisitSqlObject(ISqlVisitor visitor)
+        {
+            return visitor.Visit(this);
+        }
+    }
+
+    public enum InformationType
+    {
+        Statement,
+        Condition,
+        Combined
+    }
+
+    public class StatementInformation : Information
+    {
+        public StatementInformation(List<InformationAssignment> assignments)
```

Comment:
According to the builder, we only use `List<CompositeInformationAssignment>` , so I suggest to use the derived type. It'll be easier to extract information from the object in ou different Checkers

Reply:
Following another comment, CompositeInformationAssignment can be dropped so this comment is obsolete.